web/src/i18n/locales/en.yaml

```diff
@@ -193,31 +194,30 @@
     namePrefixAndNum: Core Activity {{number}}
     description: 
       title: Activity Overview
-      helpText:
-      reminder: >-
+      helpText: >-
         Briefly summarize activity needs
         for how requested funding will be used (e.g. administrative, outreach,
         auditing, Public Health Registry Development, etc)
+      reminder: >-
+        
       summary: 
         title: Brief Activity Summary
       detail:
         standard:
           subheader: 
         hie:
           subheader:
-          helpText:
+          helpText: >-
+           Detail linkage to meaningful use, including value proposition to providers, and a sustainability plan describing how the activity will be sustained after HIE DDI funding is exhausted. 
```

Comment:
Just poking around the YAML, do we need to define acronyms within the text somewhere? I did a search for DDI in the YAML and its never defined. Other examples like HIT, HIE, etc.

Reply:
I do have the glossary in the wiki but this is a good question that it might be worth defining acronyms, I had the sense that people would be familiar with the terms but probably not good to fiat that. I'll make a change.

Reply:
Also on CircleCI, let me go back and look at the code changes there is probably an error somewhere this happened once before one of the perils of the most recent changes is it sometimes caused these.